lib/hathifiles_database/delta.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module HathifilesDatabase
+  # A class responsible for the list of ids used to cull unnecessary updates to the hf table
+  # during a monthly update, and used for deleting entries that no longer exist.
+  # It answers the related questions:
+  # 1. Was the metadata for this HTID changed or added since the last update?
+  # 2. Was this HTID deleted since the last update?
+  # Both parameters to the initializer are text files with one HTID per line
+  # without any sorting requirements.
+  # Can be called with nil for one or both of the files and the behavior will be
+  # as if the corresponding file was empty.
+  class Delta
```

Comment:
I feel like we ought to be able to get `sort`/`comm` to do all the work rather than needing to track this... I will briefly ponder.